django/otto/management/commands/delete_old_libraries.py

```diff
@@ -0,0 +1,46 @@
+import datetime
+
+# settings
+from django.core.management.base import BaseCommand
+
+from django_extensions.management.utils import signalcommand
+
+from chat.models import Library
+
+
+class Command(BaseCommand):
+    help = "Delete libraries more than (default: 30) days old."
+
+    def add_arguments(self, parser):
+        # before date
+        parser.add_argument(
+            "--before",
+            type=str,
+            help="Delete library files older than this date. Format: YYYY-MM-DD",
```

Comment:
More clear to say "Delete libraries (and their data sources, documents, files) older than ..."